packages/nodes-base/nodes/SendGrid/SendGrid.node.ts

```diff
@@ -532,7 +532,13 @@ export class SendGrid implements INodeType {
 							enableSandbox: boolean;
 							sendAt: string;
 							headers: { details: Array<{ key: string; value: string }> };
-							attachments: string;
```

Comment:
this is potentially a breaking change, and we should create a new version of the node, leaving the old node as it is.

Reply:
Sorry, I don't know how to do this. I've updated `nodeVersion` and `codexVersion` in `SendGrid.node.json` -- is that not enough?
Could you provide more detailed instructions on how to correctly update the node version properly?